tools/ci_connector_ops/ci_connector_ops/pipelines/tests/common.py

```diff
@@ -145,6 +145,7 @@ async def _run(self) -> StepResult:
         include = [
             str(self.context.connector.code_directory),
             str(self.context.connector.documentation_file_path),
+            str(self.context.connector.migration_guide_file_path),
```

Comment:
fix: we weren't adding the migration guide file path to the context, so qa checks wouldn't notice it even if the connector dev had added it.